source/data-formats/modeling-bson-data.txt

```diff
@@ -11,140 +11,198 @@ Modeling BSON Data
    :depth: 2
    :class: singlecol
 
-.. _php-type-map:
+Overview
+--------
 
-Type Maps
----------
+In this guide, you can learn how to create and interact with BSON documents
+by using the {+driver-short+}.
 
-Most methods that read data from MongoDB support a ``typeMap`` option, which
-allows control over how BSON is converted to PHP. Additionally,
-the :phpclass:`MongoDB\Client`, :phpclass:`MongoDB\Database`, and
-:phpclass:`MongoDB\Collection` classes accept a ``typeMap`` option, which can
-be used to specify a default type map to apply to any supporting methods and
-selected classes (e.g. :phpmethod:`MongoDB\Client::getDatabase()`).
+**BSON**, or Binary JSON, is the data format that MongoDB uses to organize
+and store data. This data format includes all JSON data structure types and
+supports additional types, including dates, different-sized integers, ``ObjectId``
+values, and binary data. The {+library-short+} provides the ``MongoDB\Model\BSONArray``
+and ``MongoDB\Model\BSONDocument`` types to store BSON data.
 
-The :phpclass:`MongoDB\Client`, :phpclass:`MongoDB\Database`, and
-:phpclass:`MongoDB\Collection` classes use the following type map by
-default:
+.. tip::
 
-.. code-block:: php
+   For a complete list of supported BSON types, see :manual:`BSON Types
+   </reference/bson-types>` in the {+mdb-server+} manual.
 
-   [
-       'array' => 'MongoDB\Model\BSONArray',
-       'document' => 'MongoDB\Model\BSONDocument',
-       'root' => 'MongoDB\Model\BSONDocument',
-   ]
+.. _php-bson-sample:
 
-The type map above will convert BSON documents and arrays to
-:phpclass:`MongoDB\Model\BSONDocument` and
-:phpclass:`MongoDB\Model\BSONArray` objects, respectively. The ``root`` and
-``document`` keys are used to distinguish the top-level BSON document from
-embedded documents, respectively.
+Sample Data
+~~~~~~~~~~~
 
-A type map may specify any class that implements
-:php:`MongoDB\BSON\Unserializable <mongodb-bson-unserializable>` as well as
-``"array"``, ``"stdClass``", and ``"object"`` (``"stdClass``" and ``"object"``
-are aliases of one another).
+The code examples in this guide reference the following sample BSON document:
 
-.. seealso::
+.. code-block:: js
 
-   :php:`Deserialization from BSON <manual/en/mongodb.persistence.deserialization.php>` in the PHP manual
+   {
+       "address" : {
+           "street" : "Pizza St", 
+           "zipcode" : "10003" 
+       },
+       "coord" : [-73.982419, 41.579505]
+       "cuisine" : "Pizza",
+       "name" : "Mongo's Pizza"
```

Comment:
S: consider using a name that doesn't include a char that needs escaping, unless you specifically want to demonstrate this (which is good too!)

Reply:
Updated, I don't think it's necessary to demonstrate